### PR TITLE
Add Parameter Store Values as ENV vars

### PR DESCRIPTION
**Pull request by adhoc-nshipman**
## Summary

- Add parameter store values as ENV variables

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/94628

## Testing done

- Deployments were done to dev with the initial changes to the parent helm chart. This change continues to add more
variables to the deployments

@patrick-black-oddball

### CODE REVIEW

**Review by stevenjcumming — APPROVED** [review]

**Comment by stevenjcumming** [comment]
@adhoc-nshipman please ask Patrick for a review before you merge

**Review by Efe-Oddball — APPROVED** [review]

File: chart/values.yaml [hunk 1]
```diff
@@ -1,8 +1,79 @@
 ssmPath: /dsva-vagov/vets-api/dev/
 parameters:
-  SETTINGS__HOSTNAME: "hostname"
-  SETTINGS__VSP_ENVIRONMENT: "vsp_environment"
-  SETTINGS__VIRTUAL_HOSTS: "virtual_hosts"
```
Comment (patrick-black-oddball) [inline chart/values.yaml:5]: are these being deleted?

Comment (adhoc-nshipman) [reply chart/values.yaml:5]: Good catch @patrick-black-oddball. I readded the parameters

File: chart/values.yaml [hunk 2]
```diff
@@ -1,8 +1,79 @@
...
+  SETTINGS__DATABASE_URL: "database_url"
+  SETTINGS__TEST_DATABASE_URL: "test_database_url"
+  SETTINGS__SECRET_KEY_BASE: "secret_key_base"
+  SETTINGS__OLD_SECRET_KEY_BASE: "old_secret_key_base"
+  SETTINGS__LOCKBOX__MASTER_KEY: "lockbox/master_key"
+  SETTINGS__DMC__CLIENT_ID: "debts_api/dmc/client_id"
+  SETTINGS__DMC__CLIENT_SECRET: "debts_api/dmc/client_secret"
+  SETTINGS__DMC__URL: "debts_api/dmc/url"
+  SETTINGS__DMC__DEBTS_ENDPOINT: "debts_api/dmc/debts_endpoint"
+  SETTING__MCP__VBS__URL: "mcp/vbs/url"
```
[the remark this answers is not in the document]
Comment (adhoc-nshipman) [reply chart/values.yaml]: Confirmed with Steven that these should be SETTINGS and was able to update the variables.

**Pull request closed without merging**